src/components/PropTable.js

```diff
@@ -82,6 +86,7 @@ export default class PropTable extends React.Component {
               <td>{row.propType}</td>
```

Comment:
@arunoda I think we should work on the change in the next improvement. This shouldnt touch other implementations IMO